### PR TITLE
fix: remove legacy register page

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- We no longer support the register page, thus can be removed.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by idoshamun — APPROVED** [review]

**Comment by idoshamun** [comment]
I think there are some unused functions now, thanks to this removal

**Review by sshanzel — APPROVED** [review]
Nice cleanup!

**Comment by rebelchris** [comment]
@idoshamun / @sshanzel Removed some more unused code parts and functions

File: packages/shared/src/lib/auth.ts [hunk 1]
```diff
@@ -36,7 +36,6 @@ export enum AuthTriggers {
   LegacyLogout = 'legacy_logout',
   Verification = 'verification',
   VerifySession = 'verify session',
-  CreateFeedFilters = 'create feed filters',
```
Comment (idoshamun) [inline packages/shared/src/lib/auth.ts:39]: Is not used? So what's the trigger of the onboarding?

**Review by idoshamun — APPROVED** [review]
Deleted code everywhere! 🤩

**Review by sshanzel — CHANGES_REQUESTED** [review]
This is massive! How did you find these ones?! 🧙 

Just had one comment about something we missed when we migrated from a hook to context for the onboarding asset.

File: packages/shared/src/lib/auth.ts (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/lib/auth.ts:39]: Looks to be non tracked anymore from onboarding as it's not invoking "showLogin" but a fixed version.
It does have some other tracking events inside of it.

File: packages/shared/src/graphql/common.ts [hunk 2]
```diff
@@ -81,5 +81,4 @@ export const isQueryKeySame = (left: QueryKey, right: QueryKey): boolean => {
 
 export enum ApiError {
   Forbidden = 'FORBIDDEN',
-  NotFound = 'NOT_FOUND',
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/graphql/common.ts:84]: Hmm, nothing was using this at the moment though?
Rather re-add it when it's needed

File: packages/shared/src/hooks/useAssetPreload.ts [hunk 3]
```diff
@@ -1,19 +0,0 @@
-import { useEffect } from 'react';
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/hooks/useAssetPreload.ts:1]: No use-cases found, do you think it got removed anywhere?
If so happy to add a new task for it?  *(but looks to be no longer needed then)

File: packages/shared/src/graphql/common.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/shared/src/graphql/common.ts:84]: There are a few existing literal usage of 'NOT_FOUND', but I agree, we should probably then re-add it once needed or when we refactor those calls. Kindly disregard this comment.

File: packages/shared/src/hooks/useAssetPreload.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/shared/src/hooks/useAssetPreload.ts:1]: Yes, I believe this got removed. It is just a UX concern. If we want to support those cases with slow internet connectivity, then probably good to add a ticket for it. If not, we can just skip this through. Will leave it up to you 🙏 
https://github.com/dailydotdev/apps/blob/78ed0a85a128f0fd9b4c6fa5e196838c7c594d81/packages/shared/src/hooks/useOnboardingModal.ts#L42

**Review by sshanzel — APPROVED** [review]
Thank you for the massive effort you put into this. Really helps everyone avoid getting confused.

**Pull request merged**